tests/ccgbugs/tgc_bug.nim

```diff
@@ -0,0 +1,16 @@
+discard """
+matrix: "--gc:refc -d:fulldebug -d:smokeCycles"
+output: '''@["a"]'''
+labels: "32_bit_only codegen gc gcc seq"
```

Comment:
```suggestion
labels: "codegen gc seq"
```

The issue is not related to the C compiler or processor architecture.

Reply:
In my [defense[(https://github.com/nim-lang/Nim/issues/6279#issuecomment-324914791) :P